schema-shared.graphql

```diff
@@ -75,6 +75,11 @@ type CollectionPartnership {
   blurb: Markdown!
 }
 
+type Label {
```

Comment:
usually we add docs on what the type signifies. this docs will show up in apollo-studio for clients to read about this entity